apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceContainsRegexAssertionsSpec.kt

```diff
@@ -17,6 +18,16 @@ class CharSequenceContainsRegexAssertionsSpec : Spek({
         "◆ ", "⚬ "
     )
 
+    object RegexSpec : ch.tutteli.atrium.specs.integration.CharSequenceContainsRegexAssertionsSpec(
+        getNameContainsRegex(),
+        getAtLeastTriple(),
+        getAtLeastIgnoringCaseTriple(),
+        getShortcutTriple(),
+        getAtMostTriple(),
+        getAtMostIgnoringCaseTriple(),
```

Comment:
Those test the functionality with `String` write own functions which use `Regex` and use those